src/Vulkan/VMASharp_API_For_Review.md

```diff
@@ -0,0 +1,307 @@
+```cs
+namespace VMASharp.Metadata
+{
+  public interface IBlockMetadata
+  {
+    long Size { get; }
+    int AllocationCount { get; }
+    long SumFreeSize { get; }
+    long UnusedRangeSizeMax { get; }
+    bool IsEmpty { get; }
+    void Validate();
+    void CalcAllocationStatInfo(out StatInfo outInfo);
+    void AddPoolStats(ref PoolStats stats);
```

Comment:
What does this method do to the stats ?

Reply:
Something like [this.](https://github.com/sunkin351/VMASharp/blob/master/VMASharp/Metadata/BlockMetadata_Generic.cs#L738)